README.md

```diff
@@ -1,70 +1,98 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hotel Booking app Front-end
 
-## Available Scripts
+### Back-end API access on:
```

Comment:
Maybe put more here about connecting to API, lead them to the service duck file where they could change port if necessary etc.